runtimes/standalone/src/lib.rs

```diff
@@ -389,6 +389,8 @@ parameter_types! {
 	pub const MaxTotalKeyAgreementKeys: u32 = 1000;
 	#[derive(Debug, Clone, PartialEq)]
 	pub const MaxEndpointUrlsCount: u32 = 3u32;
+	// 600 blocks * 6s/block = 3.600s = 1h
+	pub const MaxBlocksTxValidity: u64 = 600u64;
```

Comment:
W: I see why you might have not added it to the primitives as I requested above. I would still do that for `peregrine` and soon `spiritnet` as well as keep this definition here separate for `standalone`. 

In any case, please use `BlockNumber` type.

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/257/commits/c471802786366c73e670f33afc75d8498d1a0758